support/ci/proxy_ci.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Add a local user for kitchen with a password we control.
+sudo -E useradd $MACHINE_USER --shell /bin/bash --create-home
+sudo -E usermod -p `openssl passwd -1 $MACHINE_PASS` $MACHINE_USER
+sudo -E usermod -aG sudo $MACHINE_USER
+
+# Make sure SSH can run on Azure DevOps hosted agent.
+sudo mkdir -p /var/run/sshd
+sudo service ssh restart
+
+# Install squid and git.
+sudo apt-get update
+sudo apt-get -y install squid3 git curl 
+git clone https://github.com/smurawski/proxy_tests.git
```

Comment:
Is this a permanent location?

Reply:
I had to make some updates to that repo given how the package installs on the build agent.  I'd be happy to push the changes back upstream, but I'm not sure what else depends on it.